src/components/KPrompt/KPrompt.vue

```diff
@@ -7,29 +7,28 @@
     :title="displayTitle"
   >
     <template #header-content>
-      <div class="k-prompt-header w-100">
-        <div class="k-prompt-header-content d-flex align-items-center w-100">
+      <div class="k-prompt-header">
+        <div class="k-prompt-header-content">
           <slot name="header-content">
             <KIcon
               v-if="type === 'warning'"
-              class="mr-2"
-              color="var(--white)"
+              class="warning-icon"
+              :color="`var(--white, ${KUI_COLOR_TEXT_INVERSE})`"
               icon="warning"
-              secondary-color="var(--yellow-400)"
-              size="20"
+              secondary-color="currentColor"
+              :size="KUI_ICON_SIZE_40"
             />
             {{ displayTitle }}
           </slot>
           <div class="close-button">
             <KButton
               aria-label="Close"
-              class="non-visual-button"
               @click="close"
             >
               <KIcon
-                color="var(--grey-600)"
+                :color="`var(--grey-600, ${KUI_COLOR_TEXT_NEUTRAL})`"
```

Comment:
same comment here about providing the CSS variable token

Reply:
Fixed